include/pipert/Profiler.h

```diff
@@ -78,10 +78,8 @@ class Profiler {
 
  private:
   void Move(Profiler&& o);
-  static void SendToUDP(UDPConnection* udp_connection, std::uint8_t* buffer,
-                        int buffer_size);
-  static void SendToFile(std::FILE* destination_file, std::uint8_t* buffer,
-                         int buffer_size);
+  static void SendToUDP(UDPConnection* udp_connection, std::uint8_t* buffer,int buffer_size);
```

Comment:
This is a non-functional change and because of this, it probably breaks the coding convention.